widget/ui/src/components/Modal/Modal.styles.ts

```diff
@@ -179,6 +179,10 @@ export const Content = styled('div', {
   overflowX: 'hidden',
 });
 
+export const DialogTitle = styled(Dialog.DialogTitle, {
+  margin: 0,
```

Comment:
I think this will affect a11y and they did the trick for that. We can ignore 1px difference.

Reply:
This is not to handle 1px difference. Actually with position set to `absolute`, there is no 1px difference. When we wrap `title` with `Dialog.DialogTitle` it will be wrapped in a `h2` element which have a default margin so I added this to remove the unwanted margin.